src/app/shared/url.utils.ts

```diff
@@ -0,0 +1,40 @@
+export function obj2SearchParams(obj: { [key: string]: any }) {
+  const usp = new URLSearchParams();
+  Object.keys(obj).forEach(key => {
+    usp.set(key, obj[key]);
+  });
+  return usp;
+}
+
+export function getSearchParamsHashRouting(url = window.location.href) {
+  if (url.indexOf('http') === -1) {
```

Comment:
That's right... I used this from one of our other apps.

I think `url` should then be passed to `const urlHashRouting = new URL(url, window.location.origin);`

It's just in case someone doesn't wants to use `window.location.href`.

Reply:
Thanks, that sounds reasonable. 
I'm also currently working on using those changes to tackle this item: https://github.com/riesgos/dlr-riesgos-frontend/projects/3#card-78898647

Need a little time to understand the url-api though.